tests/integration-tests/host-exports/src/mapping.ts

```diff
@@ -1,7 +1,9 @@
 import { Trigger } from "../generated/Contract/Contract";
 import { Address, BigDecimal, BigInt, ethereum } from "@graphprotocol/graph-ts";
 
-// Test that host exports work in globals.
+// TODO: remove this, graph-ts should guarantee this
+__alloc(0);
```

Comment:
Can this TODO be removed?

Reply:
Yes, this should be removed 😅 

I'll remove both TODO and `__alloc`.

The `__alloc` should be called by my `graph-ts` branch.

Reply:
Removed!